src/test/regress/isolation_schedule

```diff
@@ -1,4 +1,10 @@
+test: isolation_add_node_vs_copy_reference_table
+
+# the tests that changes node metadata should proceed
+# isolation_cluster_management so that the tests that
+# come later can be parallelized
```

Comment:
grammar nazi: "tests that change node metadata should precede isolation_cluster_management such that tests that come later can be parallelized"